CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ macro(build_spdlog)
   else()
     list(APPEND extra_cmake_args "-DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}")
   endif()
-  list(APPEND extra_cmake_args "-DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}")
+  list(APPEND extra_cmake_args "-DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS} -Wno-sign-conversion")
```

Comment:
I don't think this is going to work on Windows; you'll probably need an alternate complier flag there.